### ISSUE TITLE
Locks up system when running from docker

### ISSUE THREAD

**Issue by SLGShark6** [coturn/coturn#1257]
```
Client: Docker Engine - Community
 Version:           20.10.17
 API version:       1.41
 Go version:        go1.17.11
 Git commit:        100c701
 Built:             Mon Jun  6 23:03:11 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.17
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.11
  Git commit:       a89b842
  Built:            Mon Jun  6 23:01:29 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.8
  GitCommit:        9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

I am honestly stumped, I have tried running the docker image using the `docker run -d -p 3478:3478 -p 3478:3478/udp -p 5349:5349 -p 5349:5349/udp -p 49152-65535:49152-65535/udp coturn/coturn` command, as well as trying to run it from a compose file, but it always ends up locking up the system, forcing me to do a restart.

I first tried it on my VPS server, and just starting the image pushes the memory utilization as well as the CPU usage to 100%. At first I thought it might be because I only have 4GB of RAM on that server. But I tried on a dedicated server with 32GB of ram and a 16 core thread ripper CPU, and it did the same thing. Both these servers are running AlmaLinux 8. However I even tried on my local Debian instance and would you believe it, same thing.

Running top/htop doesn't show anything for the utilization, and this only happens with this image, I am running several other docker images (vaultwarden, mailcow, rustdesk, etc.) and have no issue.

Your assistance is appreciated, maybe I'm doing something wrong. Thank you.

**Comment by eakraly** [comment]
Hi @SLGShark6 ! That is unusual and unexpected.
Can you capture any logs from the start?

**Comment by SLGShark6** [comment]
So from my testing, it seems it has something to do with `49152-65535:49152-65535/udp` port range, as I've run it without it and it seems to startup and run fine.

I cannot get any logs to generate with that port range included, the system just immediately locks up.

Logs WITHOUT the port range included:
```
coturn-Coturn-1  | 0: (1): INFO: System cpu num is 4
coturn-Coturn-1  | 0: (1): INFO: System cpu num is 4
coturn-Coturn-1  | 0: (1): INFO: System enable num is 4
coturn-Coturn-1  | 0: (1): WARNING: Cannot find config file: turnserver.conf. Default and command-line settings will be used.
coturn-Coturn-1  | 0: (1): INFO: Coturn Version Coturn-4.6.2 'Gorst'
coturn-Coturn-1  | 0: (1): INFO: Max number of open files/sockets allowed for this process: 1048576
coturn-Coturn-1  | 0: (1): INFO: Due to the open files/sockets limitation, max supported number of TURN Sessions possible is: 524000 (approximately)
coturn-Coturn-1  | 0: (1): INFO:
coturn-Coturn-1  |
coturn-Coturn-1  | ==== Show him the instruments, Practical Frost: ====
coturn-Coturn-1  |
coturn-Coturn-1  | 0: (1): INFO: OpenSSL compile-time version: OpenSSL 3.0.9 30 May 2023 (0x30000090)
coturn-Coturn-1  | 0: (1): INFO: TLS 1.3 supported
coturn-Coturn-1  | 0: (1): INFO: DTLS 1.2 supported
coturn-Coturn-1  | 0: (1): INFO: TURN/STUN ALPN supported
coturn-Coturn-1  | 0: (1): INFO: Third-party authorization (oAuth) supported
coturn-Coturn-1  | 0: (1): INFO: GCM (AEAD) supported
coturn-Coturn-1  | 0: (1): INFO: SQLite supported, default database location is /var/lib/coturn/turndb
coturn-Coturn-1  | 0: (1): INFO: Redis supported
coturn-Coturn-1  | 0: (1): INFO: PostgreSQL supported
coturn-Coturn-1  | 0: (1): INFO: MySQL supported
coturn-Coturn-1  | 0: (1): INFO: MongoDB supported
coturn-Coturn-1  | 0: (1): INFO: Default Net Engine version: 3 (UDP thread per CPU core)
coturn-Coturn-1  | 0: (1): INFO: Domain name: a2noc.net
coturn-Coturn-1  | 0: (1): INFO: Default realm: a2noc.net
coturn-Coturn-1  | 0: (1): ERROR: CONFIG: Empty cli-password, and so telnet cli interface is disabled! Please set a non empty cli-password!
coturn-Coturn-1  | 0: (1): WARNING: cannot find certificate file: turn_server_cert.pem (1)
coturn-Coturn-1  | 0: (1): WARNING: cannot start TLS and DTLS listeners because certificate file is not set properly
coturn-Coturn-1  | 0: (1): WARNING: cannot find private key file: turn_server_pkey.pem (1)
coturn-Coturn-1  | 0: (1): WARNING: cannot start TLS and DTLS listeners because private key file is not set properly
coturn-Coturn-1  | 0: (1): INFO: Certificate file found: //turn_server_cert.pem
coturn-Coturn-1  | 0: (1): INFO: Private key file found: //turn_server_pkey.pem
coturn-Coturn-1  | 0: (1): WARNING: NO EXPLICIT LISTENER ADDRESS(ES) ARE CONFIGURED
coturn-Coturn-1  | 0: (1): INFO: ===========Discovering listener addresses: =========
coturn-Coturn-1  | 0: (1): INFO: Listener address to use: 127.0.0.1
coturn-Coturn-1  | 0: (1): INFO: Listener address to use: 172.31.0.2
coturn-Coturn-1  | 0: (1): INFO: =====================================================
coturn-Coturn-1  | 0: (1): INFO: Total: 1 'real' addresses discovered
coturn-Coturn-1  | 0: (1): INFO: =====================================================
coturn-Coturn-1  | 0: (1): WARNING: NO EXPLICIT RELAY ADDRESS(ES) ARE CONFIGURED
coturn-Coturn-1  | 0: (1): INFO: ===========Discovering relay addresses: =============
coturn-Coturn-1  | 0: (1): INFO: Relay address to use: 172.31.0.2
coturn-Coturn-1  | 0: (1): INFO: =====================================================
coturn-Coturn-1  | 0: (1): INFO: Total: 1 relay addresses discovered
coturn-Coturn-1  | 0: (1): INFO: =====================================================
coturn-Coturn-1  | 0: (1): WARNING: Cannot create pid file: /var/run/turnserver.pid
coturn-Coturn-1  | 0: (1): INFO: pid file created: /var/tmp/turnserver.pid
coturn-Coturn-1  | 0: (1): INFO: IO method: epoll (with changelist)
coturn-Coturn-1  | 0: (1): WARNING: STUN CHANGE_REQUEST not supported: only one IP address is provided
coturn-Coturn-1  | 0: (1): INFO: Wait for relay ports initialization...
coturn-Coturn-1  | 0: (1): INFO:   relay 172.31.0.2 initialization...
coturn-Coturn-1  | 0: (1): INFO:   relay 172.31.0.2 initialization done
coturn-Coturn-1  | 0: (1): INFO: Relay ports initialization done
coturn-Coturn-1  | 0: (1): INFO: Total General servers: 4
coturn-Coturn-1  | 1: (28): DEBUG: turn server id=3 created
coturn-Coturn-1  | 1: (27): DEBUG: turn server id=2 created
coturn-Coturn-1  | 1: (26): DEBUG: turn server id=1 created
coturn-Coturn-1  | 1: (25): DEBUG: turn server id=0 created
coturn-Coturn-1  | 1: (1): INFO: Total auth threads: 3
coturn-Coturn-1  | 1: (1): INFO: prometheus collector disabled, not started
coturn-Coturn-1  | 690: (1): INFO: Terminating on signal 15
```

**Comment by SLGShark6** [comment]
Not sure, but seems might be related to https://github.com/moby/moby/issues/36214

**Comment by eakraly** [comment]
TL;DR large port range makes the start very slow - narrow it down.

**Comment by AjBreidenbach** [comment]
--network host also has no performance issues afaict
